### PR TITLE
DRUP8-1059 Added view for profiles

### PR DESCRIPTION
**Pull request by Ruefa**

### CODE REVIEW

File: config/install/views.view.profile.yml [hunk 1]
```diff
@@ -0,0 +1,710 @@
...
+          field: nothing
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: custom
+          label: ''
+          exclude: false
+          alter:
+            alter_text: true
+            text: "<div class=\"p-3 osu-bg-page-alt-1 rounded-bottom\">\r\n  <h2 class=\"fw-bolder\">\r\n    {{ title }}\r\n  </h2>\r\n  <p class=\"blockquote\">\r\n    {{ field_profile_pronouns }}<br/><br/>\r\n    {{ field_profile_email }}\r\n    {{ field_profile_office_phone }}\r\n    {{ field_profile_address }}\r\n  </p>\r\n</div>"
```
Comment (emerham) [inline config/install/views.view.profile.yml]: blockquote shouldn't be here as it's not a blockquote type of element. is there a specific style needed here?

**Review by emerham — CHANGES_REQUESTED** [review]
blockquotes on paragraph text should only be used when we are wanting a blockquote styled text area. our theme has a specific look for blockquotes.

File: config/install/views.view.profile.yml (hunk 1 above)
Comment (Ruefa) [reply config/install/views.view.profile.yml]: I pulled a class from bootstrap without reading what its used for. Its really not needed so I removed it.

**Review by emerham — APPROVED** [review]
LGTM

**Pull request merged**